blocks/header/header.css

```diff
@@ -1151,3 +1152,40 @@
     white-space: nowrap;
   }
 }
+
+/* Header dark */
+
+.landing-page.dark .header-wrapper .header.block {
+  background-color: #151517;
+}
+
+.landing-page.dark .header.block hr {
+  border-color: #151517;
+}
+
+.landing-page.dark .header .header-sub-nav li.active a {
+  color: white;
+}
+/* stylelint-disable-next-line no-descending-specificity */
```

Comment:
Can you please not disable this rule unless you think there's an error see https://stylelint.io/user-guide/rules/no-descending-specificity/ for details.